### DISCUSSION TITLE
登录提示：Invalid token

### DISCUSSION THREAD

**Discussion by wu529778790** [AlistGo/alist#707] [Q&A] [answered]
登录提示：Invalid token

**Comment by xhofe** [comment, marked as answer]
密码错误。

**Comment by zengbo** [comment]
密码正确还是提示 Invalid token

**Comment by anwen-anyi** [comment]
> 密码正确还是提示 Invalid token

密码肯定错误啦……哈哈看一下日志提示的密码😂

**Reply by zengbo** [reply]
当然是用的日志的密码

**Reply by zengbo** [reply]
知道问题了，用于认证的Authorization头，被OAuth 中间件重写了。

**Reply by winnie2007** [reply]
具体是怎样修改，大佬可否教学一下？

**Reply by anwen-anyi** [reply]
> 知道问题了，用于认证的Authorization头，被OAuth 中间件重写了。

你自己魔改了吗？QAQ     **（看到另外一条明白啥意思了QAQ 加粗的为修改后的）**
